test.properties

```diff
@@ -9574,7 +9585,8 @@
         Plugin Installer
 
     testray.team.object.component.names=\
-        Object
+        Object,\
+        Upgrades Objects
```

Comment:
I think this should be `Upgrades Object` to follow our naming convention. I'm not entirely opposed to renaming `Object` to `Objects` though but that would need a separate ticket and announcement to our team.